hare/haretypes.go

```diff
@@ -29,6 +29,7 @@ const (
 )
 
 const (
+	Round0 = -1
 	Round1 = 0
 	Round2 = 1
 	Round3 = 2
```

Comment:
Round 1 should equal 1, Round 2 should equal 2 etc.
You should make sure that anywhere in the code we are consistent with round numbering.

Reply:
You are right but it is a bit risky to add this change. We can easily make an error here and I don't want this risk in this  PR (or in TN-1.0).
We can open an issue to decide how we number the rounds (the change may also be in the article).

Reply:
Btw, I would prefer naming over numbering.
Round1 should be StatusRound etc (then the article doesn't care how we number the rounds and vice versa since it is impl details).
That I can do in this PR, let me know if you want this change.

Reply:
Done